library/tasks/ScatterBam.wdl

```diff
@@ -22,6 +22,6 @@ task ScatterBam {
     disks: "local-disk ${disk_size} HDD"
     cpu: 2
     memory: "7.5 GiB"
-    docker: "us.gcr.io/broad-gotc-prod/genomes-in-the-cloud:2.3.3-1513176735"
+    docker: "quay.io/humancellatlas/secondary-analysis-picard:2.20.4"
```

Comment:
Is there a reason to prefer the quay.io one over the dockerhub one?

Reply:
I picked quay to be consistent with other docker images, since it sounds like HCA mostly pushes to quay. Does dockerhub have a recent picard docker image? I had to build this one myself.

Reply:
Ah, I see, there is the dockerhub picard image -- it seems like it is build very frequently, too. I suppose the question that remains then is whether it's better to use this automatically updated image or to use a quay image that we know the results of & is stored under HCA.

Reply:
For simplicity's sake we have opted to leave this instance of the change to be on quay. I'm adding a ticket to the backlog to look into updated all of the picard image references in Skylab, and there we will consider whether to use the dockerhub automatic releases or keep releasing our own on quay.